docs/user_guide/further_reading.rst

```diff
@@ -14,12 +14,21 @@ learning algorithms can have harmful effects:
 - `Artificial Unintelligence <https://mitpress.mit.edu/books/artificial-unintelligence>`_ by Meredith Broussard
 - `Algorithms of Oppression <https://nyupress.org/9781479837243/algorithms-of-oppression/>`_ by Safiya Umoja Noble
 
+In addition, the `Gender Shades <http://gendershades.org/>`_ study is a recent example of a
+*quality of service* harm. Specifically, algorithms which were designed to classify faces as male or
+female performed worse for darker skin tones and for females - and particularly badly on
+females with dark skin.
+
 There are also papers which focus less on individual algorithms, and more on societal impacts.
 A selection of these follows:
 
 - `Big Data's Disparate Impact <https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2477899##>`_ (Barocas & Selbst, 2016)
 - `Measures and Mismeasures of Fairness <https://5harad.com/papers/fair-ml.pdf>`_ (Corbett-Davies & Goel, 2018)
 - `Fairness through Awareness <https://arxiv.org/abs/1104.3913>`_ (Dwork et al., 2011)
 - `Algorithmic Fairness from a Non-ideal Perspective <http://zacklipton.com/media/papers/fairness-non-ideal-fazelpour-lipton-2020.pdf>`_ (Fazelpour & Lipton, 2020)
-- `A Survey on Bias and Fairness in Machine Learning <https://arxiv.org/abs/1908.09635>`_ (Mehrabi et al. 2019)
-- `Fairness and Abstraction in Sociotechnical Systems <https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3265913>`_ (Selbst et al., 2018)
\ No newline at end of file
+- `A Survey on Bias and Fairness in Machine Learning <https://arxiv.org/abs/1908.09635>`_ (Mehrabi et al., 2019)
+- `Fairness and Abstraction in Sociotechnical Systems <https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3265913>`_ (Selbst et al., 2018)
+
+A paper discussing a process (using checklists) to ensure that fairness
```

Comment:
seems like there's an issue with the sentence, either the "that" in the end should be removed or something else is missing at the end.

Reply:
Urghh yes. Obviously got distracted halfway through writing that sentence :-/